src/Infrastructure/LeanCode.ClientCredentialsHandler/LeanCode.ClientCredentialsHandler.csproj

```diff
@@ -3,6 +3,12 @@
   <ItemGroup>
     <PackageReference Include="Serilog" />
     <PackageReference Include="IdentityModel" />
+
+    <!--
+        _Something_ relies on Newtonsoft.Json but on a much older version. We can bump it to our one
```

Comment:
Well, there aren't that many direct dependencies to check, and `dotnet-depends` _should_ be able to find it. Can we document the offending packages somewhere to make it easier to track when this workaround won't be needed anymore?

Reply:
Be my guest :P